ckan/tests/lib/test_base.py

```diff
@@ -508,3 +508,20 @@ def test_cache_control_while_logged_in(app):
 
     assert 'Cache-Control' in response_headers
     assert response_headers['Cache-Control'] == 'private'
+
+
+@pytest.mark.ckan_config('ckan.cache_enabled', 'true')
+@pytest.mark.ckan_config('ckan.cache_private_enabled', 'false')
+def test_cache_control_while_logged_in_private_cache_disable(app):
+    from ckan.lib.helpers import url_for
+    user = factories.User(password="correct123")
+    identity = {"login": user["name"], "password": "correct123"}
+    request_headers = {}
+
+    response = app.post(
+        url_for("user.login"), data=identity, headers=request_headers
+    )
+    response_headers = dict(response.headers)
+
+    assert 'Cache-Control' in response_headers
+    assert response_headers['Cache-Control'] == 'private'
```

Comment:
This seems like the opposite of what we want. Private cache is disabled so we should have `Cache-Control: no-cache no-store must-revalidate max-age=0`, not `Cache-Control: private`